python/ray/util/sgd/torch/distributed_torch_runner.py

```diff
@@ -220,25 +247,25 @@ def reserve_cuda_device(retries=20):
         assert isinstance(cuda_devices, str)
         cuda_device_set = set(cuda_devices.split(","))
 
-    global _dummy_actor
+    global _dummy_cuda_actor
     unused_actors = []
 
     success = False
     for _ in range(retries):
-        if _dummy_actor is None:
-            _dummy_actor = ray.remote(
-                num_gpus=1,
+        if _dummy_cuda_actor is None:
+            _dummy_cuda_actor = ray.remote(
+                num_cpus=0, num_gpus=num_gpus,
```

Comment:
here we specify the `num_cpus=0` because CPU reservation should be done by the `_dummy_cpu_actor `

Reply:
sgtm